### DISCUSSION TITLE
Why WER of Spanish is better than English?

### DISCUSSION THREAD

**Discussion by ItipatS** [openai/whisper#502] [Q&A] [answered]
Why WER of Spanish is better than English?, Can i know the detail?

**Comment by jongwook** [comment, marked as answer]
I think it's helping that the orthography of Spanish uses more direct phonetical transcriptions compared to English, despite having a smaller amount of training data.

**Comment by IlianP** [comment]
Yes and according to the following graph from the paper, it seems to be the case for Italian as well. On the other hand, the opposite seems to be true for languages such as Hebrew, Korean and Chinese (orange and green color marked by me):
![image](https://user-images.githubusercontent.com/13310932/233104460-d3fc23b9-0350-4ff5-95b1-47c446fa9c7e.png)
On the same page (p. 7), the authors also acknowledge:

> _We also observed that many of the largest outliers in terms of worse-than-expected performance according to this trend are languages that have unique scripts and are more distantly related to the Indo-European languages making up the majority of the training datasets, such as Hebrew (HE), Telugu (TE), Chinese (ZH), and Korean (KO). These differences could be due to a lack of transfer due to linguistic distance, our byte-level BPE tokenizer being a poor match for these languages, or variations in data quality._
